### DISCUSSION TITLE
blog/2024/11/mementoai-intern/

### DISCUSSION THREAD

**Discussion by devwqc** [devwqc/devwqc.github.io#68] [Blog Posts Comments]
https://devwqc.github.io/blog/2024/11/mementoai-intern/

**Comment by Seoin02** [comment]
수습 3개월도 거쳐야하는군요?? 고생많으셨고, 건승을 빕니다🍀🙏🏻🍀

**Comment by sjlim32** [comment]
한 달간 인턴 생활 고생하셨어요 :) 봉찬님과 업무가 많이 겹치지 않아 일해보지 못한 점이 아쉽지만, 그래도 가끔 사담할 때의 봉찬님의 에너지 덕분에 많이 힐링했습니다 ㅎㅎ 앞으로의 정규직 삶도 힘내시길 바라겠습니다 😁

**Comment by ShimWooJeong** [comment]
입사하셔서도 기소야와 서브웨이 길만 걸으시길 바랍니다🙇🏻‍♀️

**Comment by bamjun** [comment]
화이팅입니다 ! 그만두지만 않는다면, 수습은 그냥 패스할꺼같아요.

**Comment by wo-o29** [comment]
오 봉찬님 취업에 성공하셨군요! 축하드립니다~

**Comment by sdh20282** [comment]
배신자
